File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/examples/simple/DualInverseZ.java
package net.fwitz.math.main.binary.dual.functions.examples.simple;

import net.fwitz.math.binary.dual.DualNumber;
import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualInverseZ {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("1/z")
                .computeFn(DualNumber::inverse)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/examples/simple/DualInverseZ.kt
package net.fwitz.math.main.binary.dual.functions.examples.simple

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualInverseZ {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("1/z")
        .computeFn { z -> z.inverse }
        .render()
}